client/question/components/LearnerViewDirective.js

```diff
@@ -1180,13 +1182,46 @@ tie.directive('learnerView', [function() {
             tasks[currentTaskIndex].getId());
         };
 
+        /**
+         * Event handler to hide modal after transition animation ends.
+         */
+        $scope.hideModalAfterAnimation = function(event) {
```

Comment:
I'm not sure this needs to be defined here; it duplicates code in MonospaceDisplayModalService.